source/aggregation-pipeline-builder.txt

```diff
@@ -165,11 +193,56 @@ letter case and accent marks.
 
 To specify a custom collation:
 
-#. Click the :guilabel:`Collation` button at the top of the pipeline
-   builder.
+#. Click :guilabel:`More Options` in the top right corner of the pipeline 
+   builder. 
+
+   .. figure:: /images/compass/options-collation.png
+        :alt: Collation Option
 
 #. Enter your collation document.
 
+.. _export-agg-builder-results:
+
+Export Aggregation Results
+--------------------------
+
+To export results from your aggregation pipeline:
+
+#. Connect to the deployment containing the
+   :doc:`collection </collections>` you wish to export data from.
+
+   To learn how to connect to a deployment, see
+   :ref:`connect-run-compass`.
+
+#. Navigate to your desired collection.
+
+   You can either select the collection from the
+   :doc:`Collections </collections>` tab or click the collection 
+   in the left-hand pane. 
+
+#. Create an aggregation pipeline and run for results.
+
+   To learn how to create an aggregation pipeline, see :ref:`Create an 
+   Aggregation Pipeline <create-agg-pipeline>`. 
+
+#. Click :guilabel:`Export`.
+
+#. Choose the appropriate file type.
+
+   Under :guilabel:`Select Export File Type`, select either 
+   :guilabel:`JSON` or :guilabel:`CSV`. If you select :guilabel:`JSON`,
+   your data is exported to the target file as a comma-separated 
```

Comment:
I might have missed something, but I thought per Max's comment we were omitting comma-separated?

Reply:
Done.